examples/tv-casting-app/linux/main.cpp

```diff
@@ -218,6 +241,32 @@ void InitCommissioningFlow(intptr_t commandArg)
     }
 }
 
+CHIP_ERROR DiscoverCommissioners()
+{
+    // Send discover commissioners request
+    ReturnErrorOnFailure(gCommissionableNodeController.DiscoverCommissioners(gDiscoveryFilter));
+
+    // Give commissioners some time to respond and then ScheduleWork to initiate commissioning
+    DeviceLayer::SystemLayer().StartTimer(
```

Comment:
I believe startTimer returns a CHIP_ERROR, so this should be the final return and not ignored.

Reply:
Fixed